sympy/geometry/ellipse.py

```diff
@@ -1268,6 +1268,47 @@ def vradius(self):
         return self.args[2]
 
 
+    def second_moment_of_area(self, point=None):
+        """Returns the second moment and product moment area of an ellipse.
+
+        Parameters
+        ==========
+
+        point : Point, two-tuple of sympifiable objects, or None(default=None)
+            point is the point about which second moment of area is to be found.
+            If "point=None" it will be calculated about the axis passing through the
+            centroid of the ellipse.
+
```

Comment:
It would be ideal if you had a "Returns" section here that describes the three things the function returns.